### DISCUSSION TITLE
How to get a wallet added to WalletConnect

### DISCUSSION THREAD

**Discussion by aglawson** [WalletConnect/walletconnect-monorepo#2539] [Developer Support]
A project I'm working on will likely require some kind of Web2-login friendly wallet. I'm aware of several existing examples of that but one possible solution is to take some existing infrastructure and create our own. I just want to know what it would take to get our wallet (if we create it) to be an option on WalletConnect's modal so it could be used on secondary NFT exchanges like OpenSea. Just trying to gauge the time and effort it would take to see if it's more worthwhile to go with an existing provider or create our own.

Thanks!

**Comment by finessevanes** [comment]
Hey there - if you'd like to be added to our modal, you can follow the instructions here: https://docs.walletconnect.com/2.0/cloud/explorer#submitting-a-project
